lib/granite/rspec/satisfy_preconditions.rb

```diff
@@ -94,10 +94,12 @@ def expected_kind_of_messages_error(object, expected_kind_of_messages, message =
   end
 
   def compare(error, expected)
-    if expected.is_a? String
+    if RSpec::Matchers.is_a_matcher?(expected)
+      expected.matches?(error)
+    elsif expected.is_a?(String)
       error == expected
     else
-      error.match? expected
+      error.match?(expected)
```

Comment:
I propose to change `error` to `error_message` or just `message`.
Or, even better, to `actual`.

Reply:
Updated